src/test/compile-fail/fat-ptr-cast.rs

```diff
@@ -18,11 +18,9 @@ fn main() {
     let q = a.as_ptr();
 
     a as usize; //~ ERROR casting
-    //~^ HELP cast through a raw pointer first
     b as usize; //~ ERROR non-scalar cast
     p as usize;
     //~^ ERROR casting
-    //~^^ HELP cast through a thin pointer
```

Comment:
These should still emit the help. It's only when that `usize` is replaced with another type that it's wrong.